### DISCUSSION TITLE
Example WEBRTC with  html and javascript

### DISCUSSION THREAD

**Discussion by Jcaltamar** [bluenviron/mediamtx#1339] [Questions]
can you help me with an example of how to use WEBRTC

**Comment by JeDiE99** [comment]
~Can we send local SDP thouth API request and get SDP remote from rtsp-simple-server ?~
Explain: https://github.com/aler9/rtsp-simple-server/issues/1376

**Comment by JeDiE99** [comment]
Example: https://github.com/aler9/rtsp-simple-server/blob/main/internal/core/webrtc_index.html

**Comment by aler9** [comment]
I've added WebRTC embedding instructions into the README:

https://github.com/aler9/rtsp-simple-server#embedding-1

[a comment by an automated account is not shown]
